### PR TITLE
chore: clean up, export just two methods

### PR DESCRIPTION
**Pull request by jobo322**
the shapes supported are gaussian, lorentzian and pseudo voigt

### CODE REVIEW

**Comment by targos** [comment]
The coverage files shouldn't be in github.
- add `coverage` to the .gitignore
- run `git rm -r coverage`

File: src/shapes/sumOfLorentzians.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ */
+
+export function sumOfLorentzians(p) {
+  return function (t) {
+    let nL = p.length / 3;
+    let result = 0;
+    for (let i = 0; i < nL; i++) {
+      let p2 = Math.pow(p[i + nL * 2] / 2, 2);
+      let factor = p[i + nL] * p2;
+      result += factor / (Math.pow(t - p[i], 2) + p2);
```
Comment (josoriom) [inline src/shapes/sumOfLorentzians.js:16]: The result should be multiplied by 1 / Math.PI? Normalization purpose

**Review by josoriom — APPROVED** [review]

File: src/shapes/sumOfLorentzians.js (hunk 1 above)
Comment (jobo322) [reply src/shapes/sumOfLorentzians.js:16]: For this case, we want to find the fwhm, intensity, and position of each lorentzian of the spectrum so these Lorentzian are not normalized. Is it that you mean?

**Comment by lpatiny** [comment]
Why do we have spectrum-generator in the dependencies ?

Are the results exactly the same as before ?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,96 @@
...
+
+let nbPoints = 31;
+let xFactor = 0.1;
+let x = new Float64Array(nbPoints);
+for (let i = 0; i < nbPoints; i++) {
+  x[i] = (i - nbPoints / 2) * xFactor;
+}
+// function that receives the parameters and returns
+// a function with the independent variable as a parameter
+function sumOfLorentzians(p) {
```
Comment (lpatiny) [inline README.md:30]: Why is this function here ? If it is simply to create a function please use the 'spectrum-generator' in the example

**Pull request merged**